src/main/java/org/folio/bulkops/processor/note/CsvDownloadPreProcessor.java

```diff
@@ -53,9 +57,12 @@ protected void setConsortiaService(ConsortiaService consortiaService) {
   }
 
   public byte[] processCsvContent(byte[] input, BulkOperation bulkOperation) {
+    Map<String, UserTenant> userTenants = new HashMap<>();
     boolean isCentralOrMemberTenant = consortiaService.isCurrentTenantInConsortia(folioExecutionContext.getTenantId());
     boolean isTypeWithTenant = bulkOperation.getEntityType() == EntityType.ITEM || bulkOperation.getEntityType() == EntityType.HOLDINGS_RECORD;
```

Comment:
This can be extracted to separate method. Preferably, method should be renamed to isTypeWithTenantSupporting.